violet-framework/src/main/java/com/horstmann/violet/workspace/sidebar/graphtools/GraphToolsBarPanel.java

```diff
@@ -111,7 +111,7 @@ private GraphToolsBarButton getSelectedButton()
 
 
 
-
+/*
```

Comment:
Ofc i forgot to remove this, while I was testing something.. Gonna change it quick